$NetBSD: patch-snappy-java-1.0.5.4_Makefile,v 1.1 2021/08/13 17:25:06 wiz Exp $

It tries to download a distfile with curl. We manage this manually.

Honor LDFLAGS.

--- snappy-java-1.0.5.4/Makefile.orig	2014-07-04 05:20:00.000000000 +0000
+++ snappy-java-1.0.5.4/Makefile
@@ -20,9 +20,7 @@ $(SNAPPY_ARCHIVE):
 	@mkdir -p $(@D)
 	curl -o$@ http://snappy.googlecode.com/files/snappy-$(VERSION).tar.gz
 
-$(SNAPPY_UNPACKED): $(SNAPPY_ARCHIVE)
-	tar xvfz $< -C $(TARGET)	
-	touch $@
+$(SNAPPY_UNPACKED):
 
 jni-header: $(SRC)/org/xerial/snappy/SnappyNative.h
 
@@ -50,7 +48,7 @@ $(SNAPPY_OUT)/SnappyNative.o : $(SRC)/or
 
 
 $(SNAPPY_OUT)/$(LIBNAME): $(SNAPPY_OBJ)
-	$(CXX) $(CXXFLAGS) -o $@ $+ $(LINKFLAGS) 
+	$(CXX) $(CXXFLAGS) -o $@ $+ $(LINKFLAGS) $(LDFLAGS)
 	$(STRIP) $@
 
 clean-native: 
